apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/QueryOptions.cs

```diff
@@ -84,11 +84,6 @@ public QueryOptions() : this(new V1.ExecuteSqlRequest.Types.QueryOptions())
             return new QueryOptions(proto.Clone());
         }
 
-        /// <summary>
```

Comment:
If you don't really need the proto form, it might be worth just having the OptimizerVersion string property. You can still have a method to construct an instance from a proto if you want.
I don't much mind.

Reply:
I actually reverted this change because @larkee made me realize that merging protos is much easier than explicitly checking for each field. So leaving the protos for now but let me know if you think there's a better way to do this.